### DISCUSSION TITLE
Gravity field recovery using GRACE-FO LRI data

### DISCUSSION THREAD

**Discussion by Kyrie714** [groops-devs/groops#118] [Q&A] [answered]
Hello GROOPS Developers,

Thank you for providing such an effective and amazing software.

I noticed that the example you provided for gravity field recovery used KBR data. In order to pursue higher accuracy, I tried to use LRI data, and I used the program InstrumentResample to change the LRI data sampling from 2s to 5s, however, The result did not match the result of ITSG accurately.

I want to know if the way I processed LRI data is inappropriate, and if so, please tell me the right way.

regards
zpzeng
![2022-03degreeVariances](https://user-images.githubusercontent.com/95745305/195818331-928fe904-2f01-47b1-afc8-0e05a2c64d03.png)

**Comment by akvas** [comment, marked as answer]
Hi @sustech-zpzeng,

in our experience, the LRI data is not as straightforward to handle as KBR, so your results do not look out of the ordinary. Depending on the month, there might be missed phase-jumps, anomalies in the pre-applied laser frequency correction and timing issues. Most problems in the LRI data have been better under control in the later mission stages after mid 2020.

We found that resampling the LRI with a polynomial of degree 3-5 works best in most cases, combined with careful outlier detection using [InstrumentRemoveEpochsByCriteria](https://groops-devs.github.io/groops/html/InstrumentRemoveEpochsByCriteria.html). I would also increase the maximum degree of the estimated gravity field to 96 or higher to avoid spatial aliasing. Estimating a daily scale parameter in [parametrizationSatelliteTracking](https://groops-devs.github.io/groops/html/parametrizationSatelliteTrackingType.html) might also help.

Cheers,
Andreas

**Comment by Kyrie714** [comment]
Thank you for your advice, which is very useful.

I tried  to use Gravityfield2TimesSplines to convert AOD data to GROOPS timespline format, which is shown in the following zip file. However, the result is different from the file provided in the example. Could you please tell me which step is not suitable. If possible, can you provide an example for me.

Looking forward to your kind reply.

regards,
zpzeng

[AODprocess.zip](https://github.com/groops-devs/groops/files/9847011/AODprocess.zip)

**Comment by akvas** [comment]
Hi,

[original AOD1B files](https://isdc.gfz-potsdam.de/esmdata/aod1b/) can be converted to TimeSplines by using [GraceAod2TimeSplines](https://groops-devs.github.io/groops/html/GraceAod2TimeSplines.html). You can set a [loop](https://groops-devs.github.io/groops/html/loopType.html) over multiple input files (e.g. one month of daily AOD1B files) to generate a TimeSplines file.

Cheers,
Andreas
